docs/disable-accounts.md

```diff
@@ -0,0 +1,32 @@
+# Disable Accounts
+
+Disable and re-enable accounts in WordPress multisite at will.
```

Comment:
Would be good to see this reworded a bit to describe as a feature of Altis. It’s kind of on the terse side grammatically so something along the lines of “Altis provides the ability to…”